### PR TITLE
Custom Domain content updates

### PR DESCRIPTION
**Pull request by tamigoodall**
Custom Domain content updates
Review links:
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/auth0-managed-certificates
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/self-managed-certificates
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/additional-configuration
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/troubleshoot-custom-domains
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/set-up-cloudfront

### CODE REVIEW

File: articles/custom-domains/troubleshoot-custom-domains.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+topics:
+  - custom-domains
+  - certificates
+contentType: how-to
+useCase: 
+  - configure-customize-domains
+  - configure-auth0-managed-certificates
+---
+
+# Troubleshooting Custom Domains
```
Comment (erjohnson) [inline articles/custom-domains/troubleshoot-custom-domains.md]: should the filename just be custom-domains/troubleshoot?

File: articles/custom-domains/self-managed-certificates.md [hunk 2]
```diff
@@ -1,91 +1,130 @@
...
 Choose this option if:
 
 * You want to have more control of your certificates (such as choosing your own CA or certificate expiration)
 * You want to enable additional monitoring over your API calls to Auth0
 
-## Prerequisites
+To set up your custom domain using self-managed certificates, you need to provide your domain name to Auth0, verify that you own that domain, configure the reverse proxy, and then complete additional configuration for the Auth0 features you are using.
 
-You'll need to register and own the domain name to which you're mapping your Auth0 domain.
+<%= include('./_provide-domain-name') %>
```
Comment (joseeantonior) [inline articles/custom-domains/self-managed-certificates.md]: This is including some generic steps in order to add the domain name to Auth0. However, these generic steps (both the screenshot and the written instructions) are based on the `Auth0-Managed Certificates` option. So, on the self-managed certificates page, it's showing options to add with the a0-managed instructions.

File: config/sidebar.yml [hunk 3]
```diff
@@ -641,8 +641,22 @@ articles:
...
 
-          - title: Custom Domains with Self-Managed Certificates
+          - title: Auth0-Managed Certificates
+            url: /custom-domains/auth0-managed-certificates
+
+          - title: Self-Managed Certificates
             url: /custom-domains/self-managed-certificates
+            children:
+
+              - title: Reverse Proxy Example
```
Comment (erjohnson) [inline config/sidebar.yml]: Should probably link to custom-domains/set-up-azure-cdn + custom-domains/set-up-cloudfront

File: articles/custom-domains/troubleshoot-custom-domains.md (hunk 1 above)
Comment (tamigoodall) [reply articles/custom-domains/troubleshoot-custom-domains.md]: yeah - probably - I'll change it

File: config/sidebar.yml (hunk 3 above)
Comment (tamigoodall) [reply config/sidebar.yml]: Got it

File: articles/custom-domains/auth0-managed-certificates.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+<%= include('./_provide-domain-name') %>
+
+## Verify ownership
+
+Before you can use the domain with Auth0, you'll need to verify that you own it. 
+
+1. Go to [Dashboard > Tenant Settings](${manage_url}/#/tenant), add the CNAME verification record listed in the Dashboard to your domain's DNS record.
+
+  ![](/media/articles/custom-domains/auth0-managed.png)
```
Comment (joseeantonior) [inline articles/custom-domains/auth0-managed-certificates.md]: Last review is for this line (forgot to hit a button before)

File: articles/custom-domains/self-managed-certificates.md (hunk 2 above)
Comment (tamigoodall) [reply articles/custom-domains/self-managed-certificates.md]: Fixed

**Review by donflopez — APPROVED** [review]
🚀 🌔

File: articles/custom-domains/_provide-domain-name.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+## Provide your domain name to Auth0
+
+1. Go to [Dashboard > Tenant Settings](${manage_url}/#/tenant). 
+
+2. Select the **Custom Domains** tab.
+
+  <% if (platform === "auth0") { %>
```
Comment (machuga) [inline articles/custom-domains/_provide-domain-name.md:7]: If a customer has multiple domains enabled, what happens in this situation, @tamigoodall ?

**Review by machuga — DISMISSED** [review]

File: articles/custom-domains/_provide-domain-name.md (hunk 5 above)
Comment (tamigoodall) [reply articles/custom-domains/_provide-domain-name.md:7]: You receive an error message that states that you have reached the maximum number of custom domains allowed for one tenant.

Comment (machuga) [reply articles/custom-domains/_provide-domain-name.md:7]: We allow it in special cases and are looking at adding it in 2020. I can send you a tenant with a second domain if you need one for testing.

Comment (tamigoodall) [reply articles/custom-domains/_provide-domain-name.md:7]: They receive an error message.

Comment (machuga) [reply articles/custom-domains/_provide-domain-name.md:7]: Is this going to be a problem? How can we support their use case? They are usually large enterprise customers

**Pull request merged**